app/src/main/java/org/mozilla/focus/settings/PrivacySecuritySettingsFragment.kt

```diff
@@ -15,6 +18,13 @@ class PrivacySecuritySettingsFragment : BaseSettingsFragment(),
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         addPreferencesFromResource(R.xml.privacy_security_settings)
+
+        // Remove the biometric toggle if the software or hardware do not support it
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M || !Biometrics.hasFingerprintHardware(context)) {
+            preferenceManager.sharedPreferences.edit().remove(resources.getString(R.string.pref_key_biometric)).apply()
```

Comment:
Do you want to remove the setting altogether or just remove it from the view? From the view can just `getPreferenceScreen().removePreference(findPreference(getString(R.string.pref_key_screen)));`